pageserver/src/basebackup.rs

```diff
@@ -4,181 +4,261 @@
 //! TODO: this module has nothing to do with PostgreSQL pg_basebackup.
 //! It could use a better name.
 //!
+//! Stateless Postgres compute node is lauched by sending taball which contains on-relational data (multixacts, clog, filenodemaps, twophase files)
```

Comment:
typos: 'lauched', 'taball', 'on-relational', 'directry'

Reply:
Sorry for mistypings, fixed.